sda/service.go

```diff
@@ -72,6 +72,7 @@ type serviceFactory struct {
 	translations map[string]ServiceID
 	// Inverse mapping of ServiceId => string
 	inverseTr map[ServiceID]string
+	mutex     sync.Mutex
```

Comment:
I guess it's because of the tests then. Ok well, it does not make a huge difference and it's still a security bonus so let's go for it :)